src/xpcc/architecture/platform/devices/stm32/stm32f765_767_768_769_777_778_779-a_b_i_n_v_z-g_i.xml

```diff
@@ -0,0 +1,866 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE rca SYSTEM "../devicefile.dtd">
+<rca version="1.0">
+  <!-- WARNING: This file is generated automatically, do not edit!
+ 		Please modify the xpcc/tools/device_file_generator code instead and rebuild this file.
+ 		Be aware, that regenerated files might have a different composition due to technical reasons. -->
+  <device platform="stm32" family="f7" name="765|767|768|769|777|778|779" pin_id="a|b|i|n|v|z" size_id="g|i">
+    <flash device-size-id="g">1048576</flash>
+    <flash device-size-id="i">2097152</flash>
+    <ram>610304</ram>
+    <core>cortex-m7f</core>
+    <pin-count device-pin-id="v">100</pin-count>
+    <pin-count device-pin-id="z">144</pin-count>
+    <pin-count device-pin-id="i">176</pin-count>
+    <pin-count device-pin-id="a">180</pin-count>
+    <pin-count device-pin-id="b">208</pin-count>
+    <pin-count device-pin-id="n">216</pin-count>
+    <header>stm32f7xx.h</header>
+    <define>STM32F4XX</define>
```

Comment:
I've removed this from the device file now, if you want to use FreeRTOS on F7, please update the port to something not from the middle ages.